Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocAutoCompleteReferenceValueControlObjectTest.cs

```diff
@@ -557,6 +557,21 @@ public void TestDropDownItemDisabledWithRequiredSelection ()
       ItemDefinition GetDropDownMenuItem3 () => dropDownMenu.GetItemDefinitions().Single(x => x.ItemID == "OptCmd3");
     }
 
+    [Test]
+    public void JS_reset_ClearsBothInputs ()
+    {
+      var home = Start();
+      var bocAutoComplete = home.AutoCompletes().GetByLocalID("PartnerField_Normal");
+      var id = bocAutoComplete.GetHtmlID();
+
+      JavaScriptExecutor.GetJavaScriptExecutor(bocAutoComplete).ExecuteScript($"document.getElementById('{id}_TextValue').reset()");
+
+      var text = bocAutoComplete.GetText();
+      var hiddenValue = home.Scope.FindCss("#body_DataEditControl_PartnerField_Normal_KeyValue").Value;
+      Assert.That(text, Is.Empty);
+      Assert.That(hiddenValue, Is.EqualTo("==null=="));
+    }
+
```

Comment:
please add separate tests for autopostback and no-autopostback configurations. For the asssertions on autopostback, I think it would be best if we had the same semantics as with TestFillWith. Not sure about the no-autopostback scenario.